test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
Well first, the relationship between app and "sub-app" needs to change. Right now the sub-app referring to a sort of global App object is a bit gross. Instead "sub-apps" need to be required into the main application.

Initializers get in the way of that because you're tacking onto a global App object. The easiest migration is just to move things into the "sub-apps" `initialize` method.